app/controllers/idv/sessions_controller.rb

```diff
@@ -5,12 +5,23 @@ class SessionsController < ApplicationController
     before_action :confirm_two_factor_authenticated
 
     def destroy
-      analytics.track_event(Analytics::IDV_VERIFICATION_ATTEMPT_CANCELLED)
-      Idv::CancelVerificationAttempt.new(user: current_user).call
+      cancel_verification_attempt_if_pending_profile
+      analytics.track_event(Analytics::IDV_START_OVER, **location_params)
       user_session['idv/doc_auth'] = {}
       idv_session.clear
       user_session.delete(:decrypted_pii)
       redirect_to idv_url
     end
+
+    private
+
+    def cancel_verification_attempt_if_pending_profile
+      return if current_user.profiles.verification_pending.blank?
+      Idv::CancelVerificationAttempt.new(user: current_user).call
+    end
```

Comment:
👀 just spotting parts of this diff that touch profiles.... I *think* this is safe WRT to recovery personal key